sdk/javascript/generator/printers.py

```diff
@@ -29,9 +29,14 @@ def get_default_value(self):
 	def get_size(self):
 		return self.descriptor.size
 
-	def load(self, buffer_name='byteArray'):
+	def load(self, buffer_name='byteArray', is_aligned=False):
 		data_size = self.get_size()
-		return f'converter.bytesToInt({buffer_name}, {data_size}, {js_bool(not self.descriptor.is_unsigned)})'
+		arguments = f'{buffer_name}, {data_size}, {js_bool(not self.descriptor.is_unsigned)}'
+		# is_aligned - handles both generation of deserializeAligned for pod and enum types as long as generation of fields within struct
```

Comment:
missing ending?

Reply:
hm?

Reply:
no, I think I wanted to phrase it differently, but now IDK how ^^